### PR TITLE
MWPW-138402 removing sitemaps

### PR DESCRIPTION
**Pull request by Blainegunn**
Removing the sitemap func from home page. The SEO will generate a manual sitemap instead.
Resolves: [MWPW-138402](https://jira.corp.adobe.com/browse/MWPW-138402)

**Test URLs:**
- Before: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?martech=off
- After: https://mwpw-138402--homepage--adobecom.hlx.page/homepage/index-loggedout?martech=off

### CODE REVIEW

**Comment by Blainegunn** [comment]
@seanchoi-dev don't merge until the SEO team gives to okay. We'll also need to delete https://www.adobe.com/homepage/sitemap.xml

**Review by seanchoi-dev — APPROVED** [review]

**Review by overmyheadandbody — APPROVED** [review]
I see the sitemap was being used in a few locales, make sure you remove it from all after merging the PR

**Pull request merged**
